### ISSUE TITLE
Add .dds support

### ISSUE THREAD

**Issue by em-mw** [nomacs/nomacs#826]
I would like .dds file support. The only way that I can view these files is by opening them in [Blender](https://www.blender.org/), surprisingly Windows Media Player, and [XnViewMp](https://www.xnview.com/en/xnviewmp/).

**Comment by em-mw** [comment]
I did some more research and found out that dds is a container not an image format, sorry about that

**Comment by Onidotmoe** [comment]
I too would like .dds support. I need to use irfanview to cover that need.

**Comment by Juryki** [comment]
me too

**Comment by OliM9595** [comment]
Yes that would be great, sometimes i just want to quickly view a texture file and there is no easy way to do that

**Comment by leejuyuu** [comment]
> If you want, build this [plug-in](https://github.com/qt/qtimageformats/tree/5.15/src/plugins/imageformats/dds) and add your qdds.dll to enable viewing of the DDS files.

>  Here is the info why the plug-in is not built by default: https://github.com/qt/qtimageformats/commit/06ee5a2abc560a1041d2c9f80eaa42f5de80a4f9

_Originally posted by @novomesk in https://github.com/nomacs/nomacs/issues/987#issuecomment-2040068429_

**Comment by leejuyuu** [comment]
@Coldtrain320 I understand that you're frustrated, but please keep the discussion about DDS format in this issue. It really helps us to organise things in one place.

While it should be possible to support DDS format by manually compiling the DDS image format plugin, it seems, after a quick look, that the plugin is still in the unmaintained state when it was dropped. Therefore, I would expect extra efforts to properly support this image format, which is why we cannot simply compile and include that plugin for you.

**Comment by novomesk** [comment]
Windows user can try to use this plug-in: https://github.com/novomesk/qt-dds-image-plugin/releases/tag/v1.0.0

However, I recommend to avoid opening files from unknown origin and some features of the format may be unsupported.

**Comment by leejuyuu** [comment]
The DDS support had been enabled by default in KImageFormats by https://invent.kde.org/frameworks/kimageformats/-/commit/0b2c6d725db366e3b18770e9af8487377c5e9108. Let's close this issue now. Feel free to reopen :))
